netcode.c

```diff
@@ -4172,6 +4195,10 @@ int skillz_add_client_to_match(struct netcode_server_t * server, uint64_t skillz
         match->num_clients_in_match++;
         server->skillz_match_id[client_index] = skillz_match_id;
 
+        if( match->start_time == 0)
```

Comment:
nit spacing

Reply:
does qtCreator have code style reformatting?

Reply:
I'm looking into it.

Reply:
I mean we pretty much know CLion will do it...

Reply:
https://stackoverflow.com/questions/37597117/qt-creator-what-is-the-shortcut-to-format-code

How we define the code format I am not sure.